templates/back-office/return-receipt-content.hypr

```diff
@@ -1,60 +1,73 @@
-  <table>
-    <tr>
-        <td>
-		    <h2>{{ labels.returnReceiptHeader | string_format(customContent.companyName,model.originalOrderNumber)|safe }} </h2>
-        </td>
-    </tr>    
-    <tr>
-        <td>
-            {{ labels.returnReceiptBodyHeader | string_format(model.contact.firstName,model.contact.lastNameOrSurname,model.returnNumber,customContent.companyName,model.originalOrderNumber) |safe}}
-        </td>
-    </tr>
-    <tr>
-        <td>
-            <table class="grid">
-                <thead>
-                    <tr>
-                        <th>{{ labels.returnReceiptItemQtyColumn }}</th>
-                        <th>{{ labels.returnReceiptItemUPCColumn }}</th>
-                        <th>{{ labels.returnReceiptItemNameColumn }}</th>
-                        <th>{{ labels.returnReceiptItemTotalColumn }}</th>
-                        <th>{{ labels.returnReceiptItemStatusColumn }}</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    {% for item in model.items %}
-                        <tr>
-                            <td>
-                                {{ item.quantityReceived }}
-                            </td>
-                            <td>
-                                {{ item.product.productCode }}
-                            </td>
-                            <td>
-                                {{ item.product.name }}
-                            </td>
-                            <td>
-                                {% if item.refundAmount %}
-                                    {{ item.refundAmount }}
-                                {% endif %}
-                                {% if not item.refundAmount and model.refundAmount > model.items.ProductLossAmount %}
-                                    {{ model.refundAmount|subtract(model.items.ProductLossAmount) }}
-                                {% else %}
-                                    {{ model.refundAmount }}
-                                {% endif %}
-                            </td>
-                            <td>
-                                {{ item.receiveStatus }}
-                            </td> 
-                        </tr>
-                    {% endfor %}
-                </tbody>
-            </table>
-        </td>
-    </tr>
-    <tr>
-        <td>
-            {{ labels.returnReceiptBodyContent | string_format(customContent.websiteUrl, customContent.companyName) |safe}}
-        </td>
-    </tr>
-</table>	
+<style>
+  pre {
```

Comment:
what is for? I'm not seeing any `pre` contents

Reply:
Removed. Earlier added to test the JSON